pkg/operator/hostetcdendpointcontroller/members_test.go

```diff
@@ -24,19 +24,19 @@ func Test_getHostname(t *testing.T) {
 	}{
 		{
 			name: "valid test case for etcd member",
-			args: args{peerURLs: []string{"https://etcd-0.foouser.tests.com"}},
+			args: args{peerURLs: []string{"https://etcd-0.foouser.tests.com:2380"}},
 			want: "etcd-0",
 		},
 		{
 			name: "valid test case for etcd bootstrap node",
-			args: args{peerURLs: []string{"https://etcd-bootstrap.foouser.tests.com"}},
+			args: args{peerURLs: []string{"https://10.0.139.142:2380"}},
 			want: "etcd-bootstrap",
 		},
```

Comment:
Can we add couple of more test cases for errors? malformed url and empty peerURLs?

Reply:
sure done